password_security/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove this line on every python file. v11 is now Python 3, which only works with UTF-8 files, so there's no more need to explicitly specify it.